docs/docsite/rst/user_guide.rst

```diff
@@ -56,12 +59,12 @@ Current Defaults (always check the source code to confirm the defaults in your s
 Any of the ``Retry`` class's parameters that are not specified in the collection defaults or in your custom dictionary, are initialized using the class's defaults, with one exception: the ``raise_on_status`` parameter is always set to ``false`` unless you explicitly added it your custom dictionary. The reason is that this lets our error handling look for the expected ``hvac`` exceptions, instead of the ``Retry``-specfic exceptions. It is recommended that you don't override this as it may cause unexpected error messages on common failures if they are retried.
 
 Retry messages
```

Comment:
Tracking is not quite right, but I can't come up with anything better. To me, `Retry messages` makes sense, but it's important that it make sense to everyone else too. Does it help that this is already within the `Retrying failed requests` section? 

To help explain, this section is about the option that controls whether you see a warning message when a retry is performed (for now warn or nothing are the only options, in the future it could be something else, like logging).

Reply:
resolved over IRC